.drone.yml

```
@@ -27,26 +27,30 @@ steps:
 - name: cargo-format
   image: rust:1.64
   commands:
-  - rustup component add rustfmt &&
-    cargo fmt --all -- --check
+  - rustup component add rustfmt && cargo fmt --all -- --check
   environment:
     CARGO_HOME: /drone/src/.cargo
   depends_on:
   - cargo-deps
 
+- name: cargo-doc-ci
+  image: rust:1.64
+  commands:
+  - cargo doc --document-private-items --workspace --all-features --no-deps -- -Dwarnings
```

This will fail because of the `-- -Dwarnings`, it needs to be passed in via RUSTDOCFLAG :woman_shrugging: 
```suggestion
  - cargo doc --document-private-items --workspace --all-features --no-deps
```